code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -619,3 +619,235 @@
 	activate_pin(2)
 
 
+/obj/item/integrated_circuit/input/atmo_scanner
+	name = "integrated atmospheric analyser"
+	desc = "The same atmospheric analysis module that is integrate into every pda.  \
```

Comment:
integrated

PDA

also im not sure thats true either

Reply:
Could you elaborate? I am not sure what you mean.

Reply:
PDAs only sometimes do; not all of them do. 'every' PDA is incorrect.

Reply:
PDAs still have an atmos sensor function, and its the only portable one that auto-updates iirc.